src/app/core/components/members-dialog/members-dialog.component.ts

```diff
@@ -134,6 +134,23 @@ export class MembersDialogComponent implements OnInit, IsTabbedDialog {
   }
 
   async onAddMemberClick() {
+    const archive = this.accountService.getArchive();
+    if (
+      archive.accessRole !== 'access.role.manager' && archive.accessRole !== 'access.role.owner'
```

Comment:
Since this is also coming up on the back-end (see https://github.com/PermanentOrg/back-end/pull/80#discussion_r846524903), this is top of mind for me: does the web-app have a mapping of roles to permissions? Ideally this check would not be directly examining the role, but instead be looking up the list of permission the current user's role has and checking to see if the required permission is available.

Reply:
It doesn't! That definitely crossed my mind while writing this code, though, and I think it'd be very good to add that, though there are other explicit permissions checks throughout the application that would also need to be changed.

Reply:
```sh
$ git grep 'access.role' src/app | wc -l
99
```

Yeah, good point, that's out of scope for this PR! Probably worth spending some time thinking about how we'd want this to work, including from the angle of SDK. (cc @slifty)